csharp_dotnetcore/3.Welcome-Users/Startup.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.BotFramework;
+using Microsoft.Bot.Builder.Integration;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using WelcomeUser.State;
+
+namespace WelcomeUser
+{
+    /// <summary>
+    /// The Startup class configures services and the app's request pipeline.
+    /// </summary>
+    public class Startup
+    {
+        /// <summary>
+        /// Gets the configuration that represents a set of key/value application configuration properties.
+        /// </summary>
+        /// <value>
+        /// The IConfiguration that represents a set of key/value application configuration properties.
+        /// </value>
+        public IConfiguration Configuration { get; }
+
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Startup"/> class.
+        /// ASP.NET Core Application Statup code. This method gets called by the runtime. 
+        /// Use this method to add services to the container.
+        /// For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
```

Comment:
/// For more information on how to configure your application, visit <see ref="https://docs.microsoft.com/en-us/aspnet/core/fundamentals/startup"/>.